### DISCUSSION TITLE
Is there a way to hide certain blocks of time?

### DISCUSSION THREAD

**Discussion by mrodz** [vkurko/calendar#263] [Q&A] [answered]
Hello all!

I am building an application where users are likely to create events in the middle of the day through the afternoon and unlikely to create events in the middle of the night.

Is there any way to hide the calendar's displayed time range, e.g. from the middle of the night to the morning?

Perhaps an option like [`hiddenDays`](https://github.com/vkurko/calendar?tab=readme-ov-file#hiddendays), but `hiddenTimes`.

A lot of the space in my UI is wasted, and I would love to know if the library has a way to do this! Thank you :D

![image](https://github.com/vkurko/calendar/assets/79176075/eb9ee82c-f16b-417a-993c-2a36398e9946)

**Comment by vkurko** [comment, marked as answer]
Sure, there are options [slotMinTime](https://github.com/vkurko/calendar?tab=readme-ov-file#slotmintime) and [slotMaxTime](https://github.com/vkurko/calendar?tab=readme-ov-file#slotmaxtime) for this, which set the first and last slot to be displayed.

**Comment by mrodz** [comment]
Oh goodness, I feel so silly—Sorry! I had looked through the entire list of options and was stuck on this for many hours, but I guess I skipped over those two options. Thank you for the swift reply! Your library is awesome.
